techniques/about.html

```diff
@@ -49,73 +49,73 @@
+                      <li><a href="#changelog">変更履歴</a></li>
+                      <li><a href="#acknowledgements">謝辞</a></li>
                     </ul>
                 </div>
             </nav>
         </aside>
         
         <main id="main" class="main-content standalone-resource__main">
-           <h1>About WCAG Techniques</h1>
+           <h1>WCAG テクニック集について</h1>
               <aside class="box">
-                <header class="box-h ">Summary</header>
+                <header class="box-h ">概要</header>
                 <div class="box-i">
-                  <p><em>Techniques</em> are examples of ways to meet Web Content Accessibility Guidelines (WCAG). They are not required to meet WCAG. To start learning about WCAG, see the <a href="https://www.w3.org/WAI/intro/wcag">WCAG Overview</a>. For important information about techniques, see the <a class="understanding" href="../Understanding/understanding-techniques">Understanding Techniques for WCAG Success Criteria</a> section of Understanding WCAG.</p>
+                  <p><em>テクニック集</em>は、Web Content Accessibility Guidelines (WCAG) を満たす方法の例である。WCAG を満たすための必須要件ではない。WCAG について学び始めるには、<a href="https://www.w3.org/WAI/intro/wcag">WCAG Overview</a> を参照。テクニックに関する重要な情報は、WCAG 解説書の <a href="https://waic.jp/translations/WCAG22/Understanding/understanding-techniques">WCAG 達成基準のテクニックを理解する</a> のセクションを参照。</p>
                 </div>
               </aside>
           <section id="aboutwcag">
-              <h2>About WCAG</h2>
-              <p>Web Content Accessibility Guidelines (WCAG) provides requirements for making websites, applications, and other digital content accessible to people with disabilities. For an introduction to WCAG, supporting technical documents, and educational material, see <a href="https://www.w3.org/WAI/standards-guidelines/wcag/">WCAG 2 Overview</a>.</p>
+              <h2>WCAG について</h2>
+              <p>Web Content Accessibility Guidelines (WCAG) は、ウェブサイト、アプリケーション、及びその他のデジタルコンテンツを、障害のある人にとってアクセシブルにするための要件を提供している。WCAG の概要、関連する技術文書、教材については、<a href="https://www.w3.org/WAI/standards-guidelines/wcag/">WCAG 2 Overview</a> を参照。</p>
 
-          <p>There are additional resources that help you understand and implement WCAG. These Techniques are one type of resource. Others are explained in the <a href="https://www.w3.org/WAI/standards-guidelines/wcag/docs/">WCAG 2 Documents</a>.</p>
+          <p>WCAG を理解し実践するのに役立つ追加のリソースもある。このテクニック集は、そのリソースの一種である。その他のリソースについては <a href="https://www.w3.org/WAI/standards-guidelines/wcag/docs/">WCAG 2 Documents</a> で説明されている。</p>
           </section>
           <section id="introduction">
-            <h2>About Techniques</h2>
-              <p>The techniques provide guidance for web content  authors and evaluators on meeting WCAG success criteria. Techniques are examples of ways to meet WCAG.</p>
+            <h2>テクニック集について</h2>
+              <p>テクニック集は、ウェブコンテンツ制作者及び評価者が WCAG 達成基準を満たすにあたってのガイダンスを提供するものである。テクニック集は、WCAG を満たす方法の例である。</p>
```

Comment:
「評価者が WCAG 達成基準を満たす」というのは少し変かと。provide guidance for... なので「評価者に...提供する」ですね。こんな感じでしょうか。

```suggestion
              <p>テクニック集は、ウェブコンテンツ制作者及び評価者に対し、WCAG 達成基準を満たすにあたってのガイダンスを提供するものである。テクニック集は、WCAG を満たす方法の例である。</p>
```